docs/pages/workflow/expo-cli.md

```diff
@@ -5,1083 +5,284 @@ maxHeadingDepth: 4
-</summary>
-<p>
-
-| Option            | Description                                                   |
-| ----------------- | ------------------------------------------------------------- |
-| `--config [file]` | Deprecated: Use app.config.js to switch config files instead. |
-
-</p>
-</details>
-
-<details>
-<summary>
-<h4>expo webhooks:add</h4>
-<p>Add a webhook to a project</p>
-</summary>
-<p>
-
-| Option                 | Description                                                                                             |
-| ---------------------- | ------------------------------------------------------------------------------------------------------- |
-| `--url [url]`          | URL to request. (Required)                                                                              |
-| `--event [event-type]` | Event type that triggers the webhook. [build] (Required)                                                |
-| `--secret [secret]`    | Secret used to create a hash signature of the request payload, provided in the 'Expo-Signature' header. |
-| `--config [file]`      | Deprecated: Use app.config.js to switch config files instead.                                           |
-
-</p>
-</details>
-
-<details>
-<summary>
-<h4>expo webhooks:remove</h4>
-<p>Delete a webhook</p>
-</summary>
-<p>
-
-| Option            | Description                                                   |
-| ----------------- | ------------------------------------------------------------- |
-| `--id [id]`       | ID of the webhook to remove.                                  |
-| `--config [file]` | Deprecated: Use app.config.js to switch config files instead. |
-
-</p>
-</details>
-
-<details>
-<summary>
-<h4>expo webhooks:update</h4>
-<p>Update an existing webhook</p>
-</summary>
-<p>
-
-| Option                 | Description                                                                                             |
-| ---------------------- | ------------------------------------------------------------------------------------------------------- |
-| `--id [id]`            | ID of the webhook to update.                                                                            |
-| `--url [url]`          | URL the webhook will request.                                                                           |
-| `--event [event-type]` | Event type that triggers the webhook. [build]                                                           |
-| `--secret [secret]`    | Secret used to create a hash signature of the request payload, provided in the 'Expo-Signature' header. |
-| `--config [file]`      | Deprecated: Use app.config.js to switch config files instead.                                           |
-
-</p>
-</details>
+This is useful because you can set native breakpoints and profile any part of the application, just be sure to track changes in source control (git) in case you need to regenerate the native app with `npx expo prebuild -p ios --clean`.
```

Comment:
```suggestion
This is useful because you can set native breakpoints and profile any part of the application. Be sure to track changes in source control (git) in case you need to regenerate the native app with `npx expo prebuild -p ios --clean`.
```